app/javascript/widget/App.vue

```diff
@@ -185,6 +186,8 @@ export default {
           this.showUnreadView = true;
         } else if (message.event === 'unset-unread-view') {
           this.showUnreadView = false;
+        } else if (message.event === 'toggle-open') {
+          this.toggleOpen();
```

Comment:
@koudshoorn Curious to know why this has been added, is it this only used to focus input?